android/src/main/java/com/swmansion/gesturehandler/core/NativeViewGestureHandler.kt

```diff
@@ -99,7 +99,7 @@ class NativeViewGestureHandler : GestureHandler<NativeViewGestureHandler>() {
       if (state == STATE_UNDETERMINED && !hook.canBegin(event)) {
         cancel()
       } else {
-        view.onTouchEvent(event)
+        view.dispatchTouchEvent(event)
```

Comment:
Does this impact any other views? `dispatchTouchEvent` sounds like it does something significantly different than `onTouchEvent`.

Reply:
From my long journey through Android source code I can say that it is not entirely different. In short `ViewGroup.dispatchTouchEvent` calls `View.dispatchTouchEvent`, which calls `View. performOnTouchCallback`, which calls `View.onTouchEvent`. So in the end we still call this function. The thing is, beside calling `onTouchEvent` indirectly, android also checks for interceptions and extracts touch target.

- [ViewGroup.java](https://android.googlesource.com/platform/frameworks/base/+/master/core/java/android/view/ViewGroup.java)
- [View.java](https://android.googlesource.com/platform/frameworks/base/+/master/core/java/android/view/View.java)

Reply:
I'll go through example app one more time and focus more on the examples that use `NativeViewGestureHandler`.

Also, what exactly do you mean by 

> Does this impact any other views? 

What do you think could be affected?